src/DynamoCoreWpf/Views/Core/AnnotationView.xaml.cs

```diff
@@ -25,33 +38,242 @@ namespace Dynamo.Nodes
+        private static readonly ConnectionStateToBrushConverter _connectionStateToBrushConverter = SharedDictionaryManager.DynamoConvertersDictionary["ConnectionStateToBrushConverter"] as ConnectionStateToBrushConverter;
+        private static readonly ConnectionStateToVisibilityCollapsedConverter _connectionStateToVisibilityCollapsedConverter = new ConnectionStateToVisibilityCollapsedConverter();
+        private static readonly BoolToVisibilityCollapsedConverter _boolToVisibilityCollapsedConverter = new BoolToVisibilityCollapsedConverter();
+        private static readonly AnnotationTextConverter _annotationTextConverter = SharedDictionaryManager.DynamoConvertersDictionary["AnnotationTextConverter"] as AnnotationTextConverter;
+        private static readonly TextForegroundSaturationColorConverter _textForegroundSaturationColorConverter = SharedDictionaryManager.DynamoConvertersDictionary["TextForegroundSaturationColorConverter"] as TextForegroundSaturationColorConverter;
+        private static readonly GroupTitleVisibilityConverter _groupTitleVisibilityConverter = SharedDictionaryManager.DynamoConvertersDictionary["GroupTitleVisibilityConverter"] as GroupTitleVisibilityConverter;
+        private static readonly InverseBooleanToVisibilityCollapsedConverter _inverseBooleanToVisibilityCollapsedConverter = new InverseBooleanToVisibilityCollapsedConverter();
+        private static readonly InverseBooleanConverter _inverseBooleanConverter = new InverseBooleanConverter();
+        private static readonly NestedGroupsLabelConverter _nestedGroupsLabelConverter = new NestedGroupsLabelConverter();
+        private static readonly CollectionHasMoreThanNItemsToBoolConverter _collectionHasMoreThanNItemsToBoolConverter = new CollectionHasMoreThanNItemsToBoolConverter();
+        private static readonly BackgroundConditionEvaluator _backgroundConditionEvaluator = new BackgroundConditionEvaluator();
+        private static readonly ColorToSolidColorBrushConverter _colorToSolidColorBrushConverter = new ColorToSolidColorBrushConverter();
+        private static readonly MenuItemCheckConverter _menuItemCheckConverter = new MenuItemCheckConverter();
+        private static readonly StringToBrushColorConverter _stringToBrushColorConverter = new StringToBrushColorConverter();
+
+        //Styles
+        private static readonly Style _contextMenuSeparatorStyle = SharedDictionaryManager.DynamoModernDictionary["ContextMenuSeparatorStyle"] as Style;
+        private static readonly Style _contextMenuItemFixedWidthStyle = SharedDictionaryManager.DynamoModernDictionary["ContextMenuItemFixedWidthStyle"] as Style;
+        private static readonly Style _menuItemGroupStyle = SharedDictionaryManager.DynamoModernDictionary["MenuItemGroupStyle"] as Style;
+        private static readonly Style _contextMenuStyle = SharedDictionaryManager.DynamoModernDictionary["ContextMenuStyle"] as Style;
```

Comment:
By doing thisI guess there would be no loading of those style resources all the time?

Reply:
yes